packages/react-ui-map/src/Mapbox/index.js

```diff
@@ -28,6 +49,21 @@ export default class Mapbox extends PureComponent {
       zoom: this.props.zoom,
       theme: this.props.theme
     })
+
+    map.on('load', () => {
+      if(this.state.map !== map)
+        this.setState(map)
```

Comment:
Per our styleguide, please add a whitespace after the `if` and use brackets around the condition body.